src/components/EventDetail.js

```diff
@@ -134,19 +140,37 @@ const styles = StyleSheet.create({
   },
   stepContainer: {
     flex: 1,
-    justifyContent: 'space-around',
     alignItems: 'center',
-    backgroundColor: 'orange',
+    justifyContent: 'center',
+  },
+  labelRow: {
+    flexDirection: 'row',
+    marginTop: -21,
+  },
+      eventInfos: {
+        width: 28,
+        height: 28,
+        marginHorizontal: 10,
+        justifyContent: 'center',
+        backgroundColor: 'black',
+        color: 'white',
+        textAlign: 'center',
+      },
+  stepStatus: {
+    flex: 0,
```

Comment:
When the height is set static, do you really need `flex: 0` or can't it be removed?